docs/source/api/index.rst

```diff
@@ -1,6 +1,6 @@
 .. raw:: html
 
-    <div id="banner"><a href="https://github.com/jcbrand/converse.js/blob/master/docs/source/theming.rst">Edit me on GitHub</a></div>
+    <div id="banner"><a href="https://github.com/jcbrand/converse.js/blob/master/docs/source/index.rst">Edit me on GitHub</a></div>
```

Comment:
Looks like `api` is missing. Should be `docs/source/api/index.rst`

Reply:
will fix / force push.